docker/service/start_service.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# start_service.sh
+# Start a full Codalab Worksheets Server service
+
+set -a
+
+usage()
+{
+  echo "Starts a full Codalab Worksheets Server service. Optionally builds docker images for it. If not building local images, 'latest' tags used, otherwise images are built with the 'local-dev' tag and these images are used. [[-b --build: Build docker images first] [-h --help: get usage help]]"
```

Comment:
Can we wrap this?
Same comment about "Server service".
Improve the description of this usage.